src/ExpandableContainer/ExpandableContainer.less

```diff
@@ -0,0 +1,20 @@
+@import (reference) "../less/utilities";
+
+.ExpandableContainer--container {
+  background-color: @neutral_white;
+  color: @neutral_dark_gray;
+}
+
+.ExpandableContainer--content {
+  .padding--s();
+}
+
+.ExpandableContainer--title {
+  .padding--s();
+  font-weight: 600;
```

Comment:
No mixin for this?

Reply:
oh right there is text--semi-bold, but then I realized that in the mocks for the announcements sidebar, the text isn't bolded, so i'm going to remove this line and leave the font styling as a consumer decision